ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -24,14 +24,18 @@ fi
 cp "$(command -v openshift-install)" /tmp
 mkdir ${ARTIFACT_DIR}/installer
 
-echo "Installing from initial release ${RELEASE_IMAGE_LATEST}"
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
+  echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is an empty string, exiting"
+  exit 1
+fi
+
+echo "Installing from initial release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
```

Comment:
What if you just set OVERRIDE to `RELEASE_IMAGE_LATEST` in case it's empty instead of failing?